src/v2/ion/i18nTransformer.js

```diff
@@ -272,6 +268,9 @@ const getI18nKey = (i18nPath) => {
   if (I18N_OVERRIDE_MAPPINGS[i18nPath]) {
     i18nKey = I18N_OVERRIDE_MAPPINGS[i18nPath];
   }
+  if (I18N_BASE_ATTRIBUTE_ENROLL_PROFILE_MAPPINGS[i18nPath]) {
+    i18nKey = I18N_BASE_ATTRIBUTE_ENROLL_PROFILE_MAPPINGS[i18nPath];
+  }
```

Comment:
nit: You can utilize the [spread syntax](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax) to combine the `I18N_BASE_ATTRIBUTE_ENROLL_PROFILE_MAPPINGS` with `I18N_OVERRIDE_MAPPINGS`, which allows us to avoid this conditional.

```js
const I18N_OVERRIDE_MAPPINGS = {
  // Existing overrides
  ...I18N_BASE_ATTRIBUTE_ENROLL_PROFILE_MAPPINGS // enroll-profile strings
}

```